src/core_plugins/kbn_vislib_vis_types/public/controls/heatmap_options.html

```diff
@@ -109,16 +109,18 @@
         <table class="vis-editor-agg-editor-ranges form-group" ng-show="vis.params.colorsRange.length">
           <tr>
             <th>
-              <label>From</label>
+              <label id="heatmap-custom-range-from">From</label>
             </th>
             <th colspan="2">
-              <label>To</label>
+              <label id="heatmap-custom-range-to">To</label>
             </th>
           </tr>
 
           <tr ng-repeat="range in vis.params.colorsRange track by $index">
             <td>
               <input
+                aria-labelledby="heatmap-custom-range-from"
+                for="heatmap-custom-range-from"
```

Comment:
The `for` attribute doesn't exist on `input` elements. It is for `label` elements to refer to the input they are labeling (which doesn't work in this case, since it can only reference one element). So we can just remove the `for` here.